umap/static/umap/js/umap.features.js

```diff
@@ -15,21 +15,29 @@ U.FeatureMixin = {
   onCommit: function () {
     // When the layer is a remote layer, we don't want to sync the creation of the
     // points via the websocket, as the other peers will get them themselves.
-    if (this.datalayer.isRemoteLayer()) return
+    if (this.datalayer && this.datalayer.isRemoteLayer()) return
```

Comment:
```suggestion
    if (this.datalayer?.isRemoteLayer()) return
```

Does it make a difference I missed?

Reply:
Good hint :+1: